src/main/resources/tables.sql

```diff
@@ -0,0 +1,16 @@
+create table cameras
+(
+    id         BIGINT primary key,
+    nasa_id    int,
+    name       varchar(240)
+);
+
+create table pictures
+(
+    id         BIGINT primary key,
+    nasa_id    int,
+    img_src    varchar(240),
+    camera_id  int,
+    created_at varchar(240),
+    foreign key (camera_id) references cameras (id)
```

Comment:
Try to add version and configure optimistic lock

Reply:
I added the field `version` and added annotation `@Version` in entities.